gradle.properties

```diff
@@ -2,19 +2,19 @@
 org.gradle.jvmargs=-Xmx2G
 
 # Gradle Plugins
-javafx_plugin_version = 0.0.13
+javafx_plugin_version = 0.1.0
 shadow_version = 7.1.2
-extra_java_module_info_version = 1.3
+extra_java_module_info_version = 1.5
 
 # Poject Properties
 version = 0.1.0
 
 # Project Dependencies
-asm_version = 9.4
-fabric_cfr_version = 0.2.0
+asm_version = 9.6
+fabric_cfr_version = 0.2.1
 fabric_fernflower_version = 2.0.0
 procyon_version = 0.6.0
-mappingio_version = 0.3.0
-javaparser_version = 3.24.2
-javafx_version = 17.0.2
-checkstyle_version = 10.3.4
+mappingio_version = 0.4.2
```

Comment:
Might be nice to move to 0.5.0? Would be a nice place to test the beta.

Reply:
Done!